remoteappmanager/templates/home.html

```diff
@@ -17,11 +17,51 @@
   <!-- /.search form -->
 
   <!-- Sidebar Menu -->
-  <ul class="sidebar-menu">
+  <ul id="applist" class="sidebar-menu">
     <li class="header">APPLICATIONS</li>
-  </ul>
 
-  <ul class="sidebar-menu" id="applist">
+    <!--
+      We initialize this element with "display: none" to prevent displaying
+      "No applications found" and the loading spinner at the same time
+    -->
+    <li v-show="!loading && model.app_list.length === 0" style="display: none;">
```

Comment:
There's no gain to use `v-if` and `v-else` instead of two `v-show` in terms of lines of code. And my problem is that it changes the behavior of the dom elements, `v-if`will result in rendering the element only if the statement is true, `v-show`only changes the style attribute `display`of the element, so I have to change the selenium tests which relies on the style attribute